### ISSUE TITLE
Deploy POD to stage and document/address any remaining issues

### ISSUE THREAD

**Issue by corylown** [pod4lib/aggregator#670]
Relates to: #413

**Comment by thatbudakguy** [comment]
my deploy stopped at:
```rb
00:03 deploy:check:linked_files
      ERROR linked file /opt/app/pod/aggregator/shared/config/secrets.yml does not exist on pod-worker-stage-a.stanford.edu
```

presumably because we're not copying the relevant files from `shared_configs` to the worker vm. this seems to be because puppet is looking for a non-existent `pod-worker-prod` branch in `shared_configs`.

i opened a PR: https://github.com/sul-dlss/puppet/pull/7919

**Comment by thatbudakguy** [comment]
update: we still need some more changes in puppet to make the new worker nodes pull from `shared-configs`. ops is working on this and will open another PR for our review. once that's done we can proceed with the deploy.

**Comment by thatbudakguy** [comment]
deploy has now succeeded. the deploy was blocked on https://github.com/sul-dlss/operations-tasks/issues/2937, as noted in #413, but we were able to proceed by manually editing the `.bashrc` on the worker VM, as detailed in https://github.com/sul-dlss/operations-tasks/issues/2937.

**Comment by thatbudakguy** [comment]
current issue is that `agg:seed_from_api` fails because the `/tmp` volume is too small (2G) to store an upload. this volume is 196G (!) in prod. I was advised to change the temporary storage location, and opened a PR based on some [similar changes that Infra made on the ETD app](https://github.com/sul-dlss/puppet/pull/5985/files).

**Comment by cbeer** [comment]
Adding a new issue to the pile. There's no mailer daemon on the ubuntu servers by default:

```
/usr/local/rvm/rubies/ruby-3.0.3/lib/ruby/3.0.0/socket.rb:1214:in `__connect_nonblock': Network is unreachable - connect(2) for [::1]:25 (Errno::ENETUNREACH)
```
